src/main/java/mate/academy/internetshop/Main.java

```diff
@@ -13,7 +13,7 @@
 
 public class Main {
 
-    private static Injector injector = Injector.getInstance("mate.academy.internetshop");
+    private static final Injector injector = Injector.getInstance("mate.academy.internetshop");
```

Comment:
```suggestion
    private static final Injector INJECTOR = Injector.getInstance("mate.academy.internetshop");
```
якщо тепер це константа, то і назва має бути у відповідному стилі